### ISSUE TITLE
build script errors on Windows

### ISSUE THREAD

**Issue by nrc** [nrc/cargo-src#210]
Since the '\' in paths is treated as an escape. Either need to escape something or use `Path` instead of `String`

**Comment by ark-** [comment]
Still doing it for me using `rustc 1.27.0-nightly (ac3c2288f 2018-04-18)`, attempting to install using `cargo install cargo-src`.

Getting a lot of these types of errors

```
error: unknown character escape: U
 --> C:\Users\<USER>~1\AppData\Local\Temp\cargo-installhOHwkU\release\build\cargo-src-49fef487575c9978\out/lookup_static.rs:5:42
  |
5 | ".eslintignore" => Ok(include_bytes!("C:\Users\<USER>\.cargo\registry\src\github.com-1ecc6299db9ec823\cargo-src-0.1.0\static\.eslintignore")),
  |                                          ^
```

**Comment by nrc** [comment]
Should be fixed with version 0.1.1 (just published), you'll need to update.
